src/fromager/commands/build.py

```diff
@@ -170,8 +172,19 @@ def _is_wheel_built(
 
     try:
         logger.info(f"{req.name}: checking if {req} was already built")
-        sdist.resolve_prebuilt_wheel(wkctx, req, [wkctx.wheel_server_url])
-        return True
+        url, _ = sdist.resolve_prebuilt_wheel(wkctx, req, [wkctx.wheel_server_url])
+        build_tag = wkctx.settings.build_tag(req.name, dist_version)
+        wheel_filename = urlparse(url).path.rpartition("/")[-1]
```

Comment:
```suggestion
        wheel_filename = urlparse(url).path.rsplit("/", 1)[-1]
```

Reply:
Took this from the resolver. Will update it there as well